_posts/Getting Started/2020-07-14-create-flink-project-on-streaming-data-platform.md

```diff
@@ -20,48 +21,74 @@ This is a general guide for creating and setting up Flink job on Dell EMC Stream
 ![Upload-Artifact]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/upload-artifact.png)
 
-###### Option II: Using the Gradle Build Tool
-**1.** Make the Maven repo in SDP available to your development workstation.
+###### Option B: Using the Gradle Build Tool
+**I.** Make the Maven repo in SDP available to your development workstation. Use `kubectl ingress` command to get the host and IP address for your namespace repo.
 ```
-kubectl port-forward service/repo 9090:80 --namespace project-name
+root@ubuntu:~$ kubectl get ingress -n your-namespace
+NAME   HOSTS                                ADDRESS        PORTS   AGE
+repo   repo.your-namespace.host.com       11.111.11.11      80     24h
+```
+
+If necessary, you may need to make changes to your `hosts` file. In order to so, find and open the `hosts` files which location, depending on the operating system that you are using, is: 
+- Windows - `C:\Windows\System32\drivers\etc\hosts`
+- Linux - `/etc/hosts`
+
+Use [`vim`](https://www.vim.org/), [`Sublime Text`](https://www.sublimetext.com/), or any other text editors to add the host and IP address information to the `hosts` file.  
+After this step, you should find a similar entry from your `hosts` file as following.   
 ```
+root@ubuntu:~$ cat /etc/hosts
+    ...                           ...
+    ...                           ...  
+    ...                           ...          
+    ...                           ...
+11.111.11.11          repo.your-namespace.host.com
+```
+
+
+**II.** Add a publishing section to your `build.gradle` file. 
+
+The standard `maven-publish` Gradle plugin must be added to the project in addition to a credentials section. The following is an example `build.gradle` file.
+
+Make sure to change the `url` to the corresponding information you get from the above step. Depending on the port number, the url protocol is:
+- `80` : http
+- `443` : https 
+
+Then replace the default `username` and `password` with your credential information.   
+You may also need to customize the publication identity, such as **groupId**, **artifactId**, and **version**. If you publish shadow JARs, make sure to add `classifier = ""` and `zip64 true` to the `shadowJar` config. 
+
 
-**2.** Add a publishing section to your `build.gradle` file. The standard `maven-publish` Gradle plugin must be added to the project in addition to a credentials section. You may also need to customize the publication identity, such as **groupId**, **artifactId**, and **version**. If you publish shadow JARs, make sure add `classifier = ""` and `zip64 true` to the `shadowJar` config. The following is an example `build.gradle` file.
 ```
 apply plugin: "maven-publish"
 publishing {
     repositories {
         maven {
-            url = "http://localhost:9090/maven2"
+            url = "http://repo.your-namespace.host.com/maven2"
             credentials {
-            username "desdp"
-            password "password"
+            username "johndoe"
```

Comment:
It is not a good practice to embed credentials in code like this. See if you can place these properties in a single file that can be added to .gitignore. Alternatively, get the credentials from environment variables as shown here: https://github.com/pravega/flink-tools/blob/master/flink-tools/build.gradle#L111